standalone/src/test/java/io/pravega/local/PravegaEmulatorResource.java

```diff
@@ -86,6 +86,7 @@ public PravegaEmulatorResource(boolean authEnabled, boolean tlsEnabled, boolean
         }
         if (tlsEnabled) {
             emulatorBuilder.certFile(SecurityConfigDefaults.TLS_SERVER_CERT_PATH)
+                    .tlsProtocolVersion(SecurityConfigDefaults.TLS_PROTOCOL_VERSION)
```

Comment:
One thing we had spoken about is to add a new constructor to PravegaEmulatorResource, such that a non default value can be passed for tlsProtocolVersion.

This would easily allow to extend TlsEnabledInProcPravegaClusterTest to exercise a TLS1.3 only test for example.

Reply:
Added tests to verify strict mode for both TLSv1.2 and TLSv1.3 protocol versions.